docs/legacy_installation.md

```diff
@@ -0,0 +1,64 @@
+### Legacy installation (without Symfony Flex)
+
+1. Require plugin with composer:
+
+    ```bash
+    composer require sylius/paypal-plugin
+    ```
+
+1. Add plugin class and other required bundles to your `config/bundles.php`:
+
+    ```php
+    $bundles = [
+        Sylius\PayPalPlugin\SyliusPayPalPlugin::class => ['all' => true],
+    ];
+    ```
+
+1. Import configuration:
+
+    ```yaml
+    imports:
+        - { resource: '@SyliusPayPalPlugin/config/config.yaml' }
+    ```
+
+1. Import routes:
+
+    ```yaml
+    sylius_invoicing_admin:
+        resource: '@SyliusPayPalPlugin/config/admin_routes.yaml'
+        prefix: '/%sylius_admin.path_name%'
+    
+    sylius_invoicing_shop:
+        resource: '@SyliusPayPalPlugin/config/shop_routes.yaml'
+        prefix: /{_locale}
+        requirements:
+            _locale: ^[a-z]{2}(?:_[A-Z]{2})?$
+    ```
+
+1. Add `FOS\RestBundle` configuration to your `config/packages/sylius_paypal.yaml` file
```

Comment:
FOS rest configuration should be in `config/packages/fos_rest.yaml`, putting such "big" and common dependency into plugin is anti-pattern.